docs/products/mysql/howto/migrate-db-to-aiven-via-console.rst

```diff
@@ -0,0 +1,247 @@
+
+   .. image:: /images/products/mysql/migration-in-progress-mysql.png
+      :width: 700px
+      :alt: Set up migration
+
+   .. warning::
+
+      To avoid conflicts and replication issues while the migration is ongoing
+
+      * Do not write to any tables in the target database that are being processed by the migration tool.
+      * Do not change the replication configuration of the source database manually. Don't modify ``wal_level`` or reduce ``max_replication_slots``.
+      * Do not make database changes that could disrupt or prevent the connection between the source database and the target database. Do not change the source database's listen address and do not modify or enable firewalls on the databases.
+
+.. topic:: Migration attempt failed?
+
+   If you happen to get such a notification, investigate potential causes of the failure and try to fix the issues. When you're ready, trigger the migration again by selecting **Start over**.
+
+1. When the wizard communicates the completion of the migration, select one of the following:
+
+   * **Close connection** to disconnect the databases and stop the replication process if still active.
+   * **Keep replicating** if the replication is still ongoing and you want to keep the connection open for data synchronization.
+
+.. topic:: Replication mode active?
+
+   Your data has been transferred to Aiven but new data is still continuously being synced between the connected databases.
+
+.. image:: /images/products/mysql/migration-completed-mysql.png
+   :width: 700px
+   :alt: Set up migration
+
+Step 5 - close
+''''''''''''''
+
+When the wizard communicates the completion of the migration without indicating an active replication process, select **Close connection**.
+
+.. topic:: Result
+
+   All the data in your database has been transferred to Aiven.
+
+.. _start-over-mysql:
+
+Start over
+----------
+
+If you :ref:`stop a migration process <stop-migration-mysql>`, you cannot restart the same process. Still, the data already migrated is retained in the target database.
+
+.. warning::
+   
+   If you start a new migration using the same connection details when your target database is not empty, the migration tool truncates your target database and an existing data set gets overwritten with the new data set.
+
+Related reading
+---------------
+
+- :doc:`Migrate to Aiven for MySQL from an external MySQL </docs/products/mysql/howto/migrate-from-external-mysql>`
+- :doc:`About aiven-db-migrate </docs/products/postgresql/concepts/aiven-db-migrate>`
+- :doc:`Migrate to Aiven for PostgreSQL® with aiven-db-migrate </docs/products/postgresql/howto/migrate-aiven-db-migrate>`
+- :doc:`Migrate to Aiven for PostgreSQL® with pg_dump and pg_restore </docs/products/postgresql/howto/migrate-pg-dump-restore>`
+- :doc:`Migrate between PostgreSQL® instances using aiven-db-migrate in Python </docs/products/postgresql/howto/run-aiven-db-migrate-python>`
```

Comment:
If this is intentional I think those links are ok, but I was wondering why we have links to PostgreSQL here